algorithms/a3c/model.py

```diff
@@ -36,10 +60,9 @@ def normalized_columns_initializer(weights, std=1.0):
     """
 
     out = torch.randn(weights.size())
-    out *= std / torch.sqrt(out.pow(2).sum(1, keepdim=True))
+    out *= std / torch.sqrt(out.pow(2).sum(1, keepdim=True).expand_as(out))
     return out
 
-
 def weights_init(m):
     classname = m.__class__.__name__
     if classname.find('Conv') != -1:
```

Comment:
I see that you are initializing layers differently, but what are the magic numbers like 6? Can you make a docstring for the function?